impl/src/main/java/org/ehcache/internal/copy/SerializingCopier.java

```diff
@@ -33,6 +29,10 @@
   private final Serializer<T> serializer;
 
   public SerializingCopier(Serializer<T> serializer) {
+    if (serializer == null) {
+      throw new NullPointerException("A " + SerializingCopier.class + " instance requires a "
```

Comment:
Adding tests for the exceptional cases of configuring the `SerializingCopier` for non-serializable types at the cache and cache-manager levels too would be nice.